plugin/pkg/shoot/seedmanager/admission.go

```diff
@@ -165,3 +209,9 @@ func verifySeedAvailability(seed *garden.Seed) bool {
 	}
 	return false
 }
+
+func verifyNetworkDisjointedNess(seed *garden.Seed, shoot *garden.Shoot) bool {
```

Comment:
I'd rather call this `haveDisjointNetworks` or `hasSatisfyingNetworks` - verify rather sounds like a check to me that could yield an error.